packages/sdk/account/utils/Utils.ts

```diff
@@ -162,15 +163,11 @@ export function convertToFactor(percentage: number | undefined): number {
 
 export function makeInstallDataAndHash(
```

Comment:
what does this method do?

Reply:
@VGabriel45 this was your addition I believe

Reply:
This is for enableMode, will be helpful when we'll integrate it.